eng/source-build-release.yml

```diff
@@ -0,0 +1,121 @@
+      name: SetTag
+      displayName: Set tag
+    - script: |
+        set -euxo pipefail
+        tag="$(SetTag.Tag)"
+        cd $(Build.SourcesDirectory)
+        git show-ref --tags
+        if git rev-parse "$tag" >/dev/null 2>&1; then
+          echo "Tag $tag exists.";
+        else
+          echo "Tag $tag does not exist."
+          exit 1
+        fi
+      displayName: Ensure tag exists
+
+- stage: ReleaseAnnouncement
+  displayName: Release Announcement
+  dependsOn: Setup
+  jobs:
+  - job: create_release_discussion_announcement
+    displayName: Create Announcement
+    variables:
+    - name: Tag
+      value: $[ stageDependencies.Setup.SetupJob.outputs['SetTag.Tag'] ]
+    steps:
+
+    - script: |
+        set -euxo pipefail
+        query='query { repository(owner: "${{ variables.announcementOrg }}", name: "${{ variables.announcementRepo }}") { id } }'
+        repo_id=$( gh api graphql -f query="$query" --template '{{.data.repository.id}}' )
+        echo ${{ variables.announcementOrg }}/${{ variables.announcementRepo }} repo ID is ${repo_id}
+        echo "##vso[task.setvariable variable=repoId;]${repo_id}"
+      displayName: Get repo ID
+      env:
+        GH_TOKEN: $(BotAccount-dotnet-sb-bot-pat)
+
+    - script: |
+        set -euxo pipefail
+        query='query { repository(name: "${{ variables.announcementRepo }}", owner: "${{ variables.announcementOrg }}") { discussionCategories(first: 10) { edges { node { id, name } } } } }'
+        category_id=$( gh api graphql -f query="$query" --template '{{range .data.repository.discussionCategories.edges}}{{if eq .node.name "Announcements"}}{{.node.id}}{{end}}{{end}}' )
+        echo Discussion Category ID is ${category_id}
+        echo "##vso[task.setvariable variable=discussionCategoryId;]${category_id}"
+      displayName: Get announcement discussion category ID
+      env:
+        GH_TOKEN: $(BotAccount-dotnet-sb-bot-pat)
+
+    - script: |
+        set -euxo pipefail
+        echo Repo ID is $(repoId)
+        echo Discussion Category ID is $(discussionCategoryId)
+
+        # Read runtime version from release commit
+        git checkout $(Tag)
+        runtime_version=$(cat $(Build.SourcesDirectory)/eng/Version.Details.xml | grep 'Microsoft.NETCore.App.Runtime.win-x64' | grep -P -o -e '(?<=Version=").*?(?=")')
+        dotnet_version=$(echo "$runtime_version" | head -c 3)
+
+        # Set environment variables that go in the announcement template
+        export TAG=$(Tag)
+        export RUNTIME_VERSION="$runtime_version"
+        export SDK_VERSION="${{ parameters.sdkVersion }}"
+        export TAG_URL="https://github.com/dotnet/installer/releases/tag/$TAG"
+        export RELEASE_NOTES_URL="https://github.com/dotnet/core/blob/main/release-notes/$dotnet_version/$runtime_version/${{ parameters.sdkVersion }}.md"
+
+        # Switch back to current branch to read announcement template
+        git checkout -
+        title=".NET $SDK_VERSION Source-Build Released"
```

Comment:
Consider embedding the title into the template.  Possibly interpret the first line as the title or search for a 'tag' such as Title:

Reply:
Should the title also include the runtime version?

.NET March 2022 Update - .NET 6.0.4 and SDK 6.0.104 
.NET March 2022 Update - 6.0.4 / 6.0.104